editor.planx.uk/src/pages/FlowEditor/lib/store/settings.ts

```diff
@@ -33,6 +37,19 @@ export const settingsStore: StateCreator<
 
   setFlowSettings: (flowSettings) => set({ flowSettings }),
 
+  flowStatus: undefined,
```

Comment:
Should this be `"online"` by default rather than `undefined` ? 

Noticing that all existing flows on pizza are showing up as "offline" by default, which is opposite of intended behavior right? Perhaps just a data sync issue?

Reply:
Thanks for explanation here - totally agree `undefined` is correct store value now that data sync/migration is fixed in background :+1: